pkg/webhook/policy.go

```diff
@@ -206,10 +236,10 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 	return admission.ValidationResponse(true, "")
 }
 
-func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.Request) []string {
+func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.Request, ref *corev1.ObjectReference) []string {
 	var msgs []string
 	var resourceName string
-	if len(res) > 0 && *logDenies {
+	if len(res) > 0 {
```

Comment:
We should leave the `logDenies` gate here, and add an OR-ed second gate for `emitDenyEvents` (or similar), as I suspect users may want the ability to toggle off event emission from the webhook if it causes issues, and we should avoid running unneeded logic if they do to reduce latency (my understanding is deserialization is not trivial).

I'm split on whether we should enable emitting events from the webhook by default, as I don't know what it might do to our throughput numbers. Some load testing may be in order.